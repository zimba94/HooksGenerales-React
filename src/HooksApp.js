import React from 'react'

export const HooksApp = () => {
    return (
        <div>
            <h1>Hola Zimba</h1>
        </div>
    )
}
