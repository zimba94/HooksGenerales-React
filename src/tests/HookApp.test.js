import React from "react";
import { shallow } from 'enzyme';
import { HooksApp } from "../HooksApp";

describe('Pruebas en HookApp', () => {
    test('Debe mostarse correctamente', () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    }); 
});
