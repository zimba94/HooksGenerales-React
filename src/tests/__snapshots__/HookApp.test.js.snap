// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp Debe mostarse correctamente 1`] = `
<div>
  <h1>
    Hola Zimba
  </h1>
</div>
`;
